constructShortestPath.js

```diff
@@ -0,0 +1,93 @@
+/*
+For this approach, we can represent the input data as graph where the cities 
+connecting to each other can be reprsented as nodes
+
+and the roads can be represented as edges. 
+Also by looking at the input, we can represent the graph as undirected and build an adjacency list.
+Thus, for finding the shortest path between two cities, we can find the shorest path the same way we would between nodes in our graph.
+A BFS would be a good approach, since we know that using a BFS, you're guaranteed to find the shorest path, so we can go with that approach. 
+*/
```

Comment:
The unique part about this problem is that it is NOT exactly a standard BFS. You're not just asked to determine the LENGTH of the shortest path. You are supposed to return the ACTUAL PATH itself. Your explanation should talk about how you plan to do this.

In fact, probably a lot of the sentences you said above can be removed, and you should focus more on this point. In the beginning, you could've just said:
"We can model this as an undirected graph where the cities are nodes and the roads are edges. Then, we could easily find the length of the shortest path with a BFS. But, we need to find the path itself.
<insert explanation of how to find path>"

Reply:
Updated